src/main/javadoc/overview.html

```diff
@@ -0,0 +1,131 @@
+<body>
+
+<h1>Introduction</h1>
+
+The Frank!Framework offers services to integrate other software applications.
+The typical end user owns many software applications that need to communicate in
+a standardized way. This is done by connecting the individual software
+applications to an instance of the Frank!Framework.
+<p>
+When some existing software application A wants to reach another existing
+application B, then A typically accesses the integration solution
+instead. The integration solution receives the incoming message and
+transforms it to a format that B can understand. The transformed
+message is sent to B. This is a very simplified view, but it allows
+us to introduce the basic building blocks offered by the Frank!Framework.
+<p>
+The most important building blocks are listeners, pipes, senders and receivers.
+Listeners are the building blocks that accept incoming messages. Messages can
+originate from HTTP requests, the file system, the database and more, which
+explains that there are many types of listeners. Listeners are wrapped in
+receivers that add additional services about logging and transaction
+management. Pipes are used to transform messages. Each possible
+transformation of a message has its own type of pipe. Finally, senders
+write messages to the database, send them as HTTP requests, write
+them to files or send them elsewhere. Each of these possibilities
+has its own type of sender.
+<p>
+Integration solutions are typically developed by Frank developers, who
+decide how the building blocks have to be combined and configured.
+Frank developers provide this information through XML files and properties
+files. The Frank!Framework parses these XML files into networks of
+Java objects that can provide the actual integration solution. The
+Frank!Framework also allows developers to work with a graphical
+user interface, the Frank!Flow. To understand this from a user perspective, please
+read the <a href="http://frank-manual.readthedocs.io">Frank!Manual. Another
+source of user documentation is the Frank!Doc, which you can download from the
+Frank!Framework's web console.</a>.
+<p>
+The Frank!Framework is a Java application that provides a web interface.
+It runs on an application server. WeAreFrank! supports running the
+Frank!Framework on Apache Tomcat, JBoss Application Server (recently renamed to WildFly)
+or WebSphere Application Server. The web interface is typically
+used by operators who are responsible for the proper operation
+of the integration solution, or as WeAreFrank! says: the Frank.
+The Frank!Console is described in the <a href="http://frank-manual.readthedocs.io">Frank!Manual</a>.
```

Comment:
I would suggest to deep link here to the Frank!Console chapter of the manual directly

Reply:
I left out the Frank!Console because I did not know how to introduce that term. I already tell that the Frank!Framework is a web application, so I thought it confusing to name the web interface "Frank!Console". I have links to the Frank!Manual and to ibis4example to show the reader what the Frank!Framework does.